components/Login/LoginButton.module.css

```diff
@@ -0,0 +1,15 @@
+.login {
+  background-color: var(--spotify-color);
+  border-radius: 50px;
+  border: none;
+  font-family: var(--primary-font);
+  font-size: 16px;
+  display: flex;
+  align-items: center;
+  width: 315px;
+  height: 52px;
```

Comment:
Usually, you don't need a fix `width` and `height`.
Instead, give the `img` inside your `button`  a `height` relative to your `font-size`.
Now, you can adjust the `padding` of your `button`.